pytorch_pfn_extras/runtime/_runtime.py

```diff
@@ -470,9 +511,5 @@ def named_runtime_modules(
             for name, sm in module.named_children():
                 yield from named_runtime_modules(sm, name, False, recursive)
     else:
-        if first_level or recursive:
-            for sm in module.children():
-                for descendant in sm.modules():
-                    if _module_runtime_tag(descendant) is not None:
-                        raise ValueError("Runtimes cannot be nested.")
+        # nested runtime tag is ignored
```

Comment:
Was this change needed for this PR functionality to work?
(just asking 😄)

Reply:
(probably) yes. Nested runtime will be happened because of [this change](https://github.com/pfnet/pytorch-pfn-extras/pull/583/files#diff-1f8b64ccf0597e93822659c40efb4b0ff872d84a115abb5b17c31895468344b1R64-R67) (setting runtime_tag to the nested modules).

Also, If I do not set runtime_tag to the children modules,  [this testcase](https://github.com/pfnet/pytorch-pfn-extras/pull/583/files#diff-d6e0d68fc723026d0105df062ec8ac439f393788264a0492c137bbb3e54f0c00R148) will not pass.

Reply:
Thanks for the explanation!